### ISSUE TITLE
Telescopic Fruit Picker

### ISSUE THREAD

**Issue by nelsonic** [dwyl/phase-two#91]
Today on my morning foraging walk I picked and eat fresh persimmons off the tree:

![image](https://user-images.githubusercontent.com/194400/66700100-e0385e80-ece4-11e9-971c-01e7b8264d9b.png)

They were _delicious_! But there was an issue ... I could only reach the proverbial "_low hanging fruit_". 🤦‍♂ 
There were _many_ more (_even riper_) fruits higher up the tree that I had no way of reaching.

That got me thinking of how to **`make`** a telescopic picking tool ... 💭 
I searched for the keywords I thought would be relevant: "Telescopic Fruit Picker"

And the top (_non sponsored_) result was an ebay results page (_they have SEO figured out..._)
https://www.ebay.co.uk/sch/i.html?_from=R40&_nkw=Telescopic+Fruit+Picker

Thankfully, someone `else` has thought of this and has made a product:
https://www.ebay.co.uk/itm/Stainless-Steel-Button-Telescopic-Rod-Fruit-Picker-Picking-Persimmon-Tools-GN/352788224972
![image](https://user-images.githubusercontent.com/194400/66700144-5dfc6a00-ece5-11e9-9ea9-eb9550ed92e2.png)
![image](https://user-images.githubusercontent.com/194400/66700154-766c8480-ece5-11e9-9fbb-ae310ad69631.png)

£6.54 with "free" shipping seems _cheap_. 🤔 

Listing claims its made of stainless steel. Doubt it. (_for that price_) 🙄 
But the seller has 5k+ feedback 
![image](https://user-images.githubusercontent.com/194400/66700341-14ad1a00-ece7-11e9-90ec-c1ac9932dc86.png)

And ebay has a decent buyer protection policy ... so I don't think there is much risk.

Ordered one:
![fruit-picker-ordered](https://user-images.githubusercontent.com/194400/66700383-79687480-ece7-11e9-8913-912daeb9afce.png)

Now we wait ... ⏳ 
(_I will report back on it's usefulness when it arrives_) 😉

**Comment by carolrmc** [comment]
Why didn't they have these when I went fruit picking? all the good ones
stayed at the top of the tree where the sun ripened them first! you had to
be a nimble climber! Keen to know if it works. We fashioned something
similar with a large jam tin on a broom stick to pick prickly pears. but
the fruit had to be really ripe else it would be damaged in the process.

**Comment by nelsonic** [comment]
@carolrmc indeed, none of this existed before the internet (_because before we could buy **anything** online we were limited to our local shops which definitely didn't have it..._). We could have used it a couple of weeks ago when we had to climb the Fig tree to reach the ripest fruits ...

![image](https://user-images.githubusercontent.com/194400/66705450-bac84680-ed1e-11e9-9007-b9731a0b36de.png)

![image](https://user-images.githubusercontent.com/194400/66705434-6fae3380-ed1e-11e9-8fdb-152ec4037eee.png)

No doubt you climbed _many_ trees given how "country life" your youth was. 😍 
I was keen to **`make`** my own picker but I don't have anything _here_ I can use here that's long enough ...
However, I did buy some strips of wood at Leroy yesterday (_that barely fit in the car..._):
![image](https://user-images.githubusercontent.com/194400/66705533-f9aacc00-ed1f-11e9-9a50-942f004bc9e1.png)
So I _could_ use a couple of those and a wire coat hanger to make a basic DIY picker ... 💭 


Yesterday I Used the strips of wood to make rudimentary measuring sticks for levelling the hole for https://github.com/dwyl/home/issues/14
![image](https://user-images.githubusercontent.com/194400/66705580-7e95e580-ed20-11e9-8e86-b92e53669e19.png)

![image](https://user-images.githubusercontent.com/194400/66705535-0af3d880-ed20-11e9-864e-8eb0dc2efb4c.png)
The longer one in this photo is 4400mm comprised of two 2200mm lengths:
![image](https://user-images.githubusercontent.com/194400/66705541-17783100-ed20-11e9-9f26-c1bd7e04912c.png)
That's 4m of _internal_ dimension + 20cm of wall thickness on each side (_in case you're wondering_)

(_left them gluing over the weekend and will put them to good use on Monday on return to the building site..._)

Going to have to wait a few weeks for the picker to arrive.
Will let you know how effective it is. 👍 
Might be useful for you to pick the Avos from the tree of the fallow land near Glood 🤔

**Comment by carolrmc** [comment]
You need a bigger car for DIY projects😊.

That hole is enormous.
What happened to the kiwis and the lemon tree😢

I think a bean can - a large one - would work as well as a wire coat hanger
and doesn't need any bending or fiddling with, just nail it onto a long
enough stick.

Nothing like country life for robust, happy kids!

**Comment by nelsonic** [comment]
@carolrmc indeed the hole is big (_almost big enough for our project_) 📐 
the Lemon Tree and Kiwi Vines are a _long_ story ... 🙄 (_will show you when you visit_) 👍
